compose/runtime/runtime/api/desktop/runtime.api

```diff
@@ -18,6 +18,7 @@ public final class androidx/compose/runtime/ActualDesktop_desktopKt {
 
 public final class androidx/compose/runtime/ActualJvm_jvmKt {
 	public static final fun identityHashCode (Ljava/lang/Object;)I
+	public static final fun synchronized (Landroidx/compose/runtime/SynchronizedObject;Lkotlin/jvm/functions/Function0;)Ljava/lang/Object;
```

Comment:
Why do we need to place a new method in `ActualJvm_jvmKt`? I see you explicitly added `JvmName` there

Reply:
It's just copied from [AOSP](https://github.com/androidx/androidx/blob/androidx-main/compose/runtime/runtime/src/jvmMain/kotlin/androidx/compose/runtime/SynchronizedObject.jvm.kt) that is added … by you :)

Reply:
Right. It was for their CI checks. For desktop we agreed that our fork is a source of truth now, so since it doesn't break the existing API, we don't need that.

@igordmn WDYT? What name should we use in binary? Should we keep compatibility with the old AOSP variant? I see no reason for it, but at the same time it doesn't really matter

Reply:
This function is not from jvmMain\desktopMain. It is from commonMain with a different JvmName. We need to have the same state in commonMain.

Reply:
Actually, it is in `jvmMain`. I would keep `jvmMain` in sync with AOSP too though.

Reply:
Except `actual class SynchronizedObject` that we need for our binary compatibility. We can either upstream it, or move the actual to `desktopMain`. I vote for the first